src/components/Home/WeekRate.tsx

```diff
@@ -0,0 +1,144 @@
+import { Cell, Label, Pie, PieChart } from 'recharts';
+import styled from 'styled-components';
+import { WeekRateProps } from '../../types/Week/weekRateTypes';
+
+const CustomLabel = ({ viewBox, value }: WeekRateProps) => {
+  const { cx, cy } = viewBox;
+
+  return (
+    <>
+      <StyledText
+        x={cx}
+        y={cy - 20}
+        dominantBaseline="middle"
+        textAnchor="middle"
+      >
+        {value}
+      </StyledText>
+    </>
+  );
+};
+const WeekRate = () => {
+  const maxProblems = 0;
+  const solvedProblems = 0;
+
+  const percentage =
+    maxProblems && solvedProblems ? (solvedProblems / maxProblems) * 100 : 10;
```

Comment:
P2: `maxProblems && solvedProblems`가 조건으로 계속 쓰이고 있어서 하나의 플래그로 만들면 중복 코드도 줄이고 의미 파악에 도움이 될 것 같아요!
```suggestion
  const isProblemsExit = maxProblems && solvedProblems;
  const percentage =
    isProblemsExit ? (solvedProblems / maxProblems) * 100 : 10;
```

Reply:
수정하였습니다 !